packages/playground-react/pages/suspense/portal.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+
+import { Portal } from '@headlessui/react'
+import { Suspense, lazy } from 'react'
+
+function MyComponent({ children }: { children(message: string): JSX.Element }) {
+  return <>{children('test')}</>
+}
+
+let MyComponentLazy = lazy(async () => {
+  await new Promise((resolve) => setTimeout(resolve, 4000))
+
+  return { default: MyComponent }
+})
+
+export default function Index() {
+  return (
+    <div>
+      <h1>@headlessui/react & Suspense</h1>
+
+      <Portal>
```

Comment:
Can we position/style it a tiny bit? While testing I just noticed this and was super confused: 
<img width="966" alt="image" src="https://github.com/tailwindlabs/headlessui/assets/1834413/74b8d4fd-e13c-4012-bb5d-958a79e75b0c">


Didn't realise I had to scroll down 😅

Reply:
Ah yeah I was testing it the components directly. Suspense * useRouter don't work well together. But yeah I can do that.

Reply:
Updated